packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -123,10 +124,11 @@ export function useClipboardHandler() {
 			const eventDefaultPrevented = event.defaultPrevented;
 			event.preventDefault();
 
-			const isFullySelected = __unstableIsFullySelected();
 			const isSelectionMergeable = __unstableIsSelectionMergeable();
+			const handleWholeBlocks =
```

Comment:
In line with conventions, I'd go with `shouldHandleWholeBlocks` (or even `willHandle`), but this one is clear to me too. :)

Reply:
Updated to `shouldHandleWholeBlocks`. Thanks!